<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraOutputBusinessActionWidget</class>
 <widget class="QWidget" name="CameraOutputBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output ID:</string>
     </property>
     <property name="buddy">
      <cstring>relayComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="relayComboBox"/>
   </item>
   <item>
    <widget class="QWidget" name="fixedDurationWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="fixedDurationCheckBox">
        <property name="text">
         <string>Fixed duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="nx::vms::client::desktop::TimeDurationWidget" name="timeDuration">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>relayComboBox</tabstop>
 </tabstops>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::TimeDurationWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/business/time_duration_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
